package.json

```diff
@@ -24,7 +24,6 @@
   "peerDependencies": {
     "code-push": ">=1.8.0-beta",
     "cordova-plugin-file-transfer": ">=1.3.0",
-    "cordova-plugin-file": ">=3.0.0",
```

Comment:
Cordova actually looks [here](https://github.com/Microsoft/cordova-plugin-code-push/blob/master/plugin.xml#L10) when installing dependencies, so the fix for this would be to adjust the version numbers (in both `package.json` and `plugin.xml` to stay in sync, and keeping in mind what the file transfer plugin depends on) and not to remove the peer dependency. Would that work?